packages/lib/src/storybook/StoryMeta.types.ts

```diff
@@ -44,7 +44,9 @@ export interface ChromaticConfig {
  * Story Parameters
  */
 type StoryParameters<
+  /** The component type */
   T extends React.ElementType,
+  /** Any eXtra Props the story should support */
```

Comment:
nit - grammar

Reply:
This is intentional 😛 Since the generic is called `XP`